dxr/testing.py

```diff
@@ -267,11 +265,12 @@ def config_input(cls, config_dir_path):
 
     @classmethod
     def teardown_class(cls):
-        if cls.should_delete_instance:
+        if cls.stop_for_interaction:
+            print 'Not deleting instance in %s.' % cls._config_dir_path
+            make_app(cls.config()).run(host='0.0.0.0', port=8000)
```

Comment:
Can we import and call dxr.cli.serve.serve() instead, so we don't have to repeat the defaults (like 8000)? Also, we should remove the language here and in the .rst about deleting instances. There's no such thing as an instance anymore. (It used to be a folder full of HTML documents and a SQLite DB.)